From c11e98954bfe189e879463ba9844e45e8646259e Mon Sep 17 00:00:00 2001
From: Priyesh Padmavilasom <ppadmavilasom@vmware.com>
Date: Mon, 23 Mar 2020 23:42:00 +0000
Subject: [PATCH] fix error in distroverpkg lookup - change RPMTAG_NAME to
 RPMTAG_PROVIDES - required as system-release is provided by photon-release

---
 client/utils.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/client/utils.c b/client/utils.c
index 1421b5c..b58b2bf 100644
--- a/client/utils.c
+++ b/client/utils.c
@@ -342,7 +342,7 @@ TDNFRawGetPackageVersion(
         BAIL_ON_TDNF_ERROR(dwError);
     }
 
-    pIter = rpmtsInitIterator(pTS, RPMTAG_NAME, pszPkg, 0);
+    pIter = rpmtsInitIterator(pTS, RPMTAG_PROVIDES, pszPkg, 0);
     if(!pIter)
     {
         dwError = ERROR_TDNF_NO_DISTROVERPKG;
